src/utils.js

```diff
@@ -0,0 +1,20 @@
+import {toByteArray} from 'base64-js';
+
+export function decoder(encoded, options) {
+    var buffer = toByteArray(encoded);
+    var float;
+    if (options.precision === '64') {
+        float = new Float64Array(buffer.buffer);
+    } else {
+        float = new Float32Array(buffer.buffer);
+    }
+    return Array.from(float);
```

Comment:
Currently is not supported on chromatogram. But indeed, in the merge step I could use them as that